### ISSUE TITLE
"Buttons" on landing page have focus styles that fail WCAG 1.4.11 &  2.4.11 

### ISSUE THREAD

**Issue by mdarrik** [toastdotdev/www.toast.dev#1]
Currently the buttons on the landing page have really faint focus outlines. For instance: 
![green button from toast.dev, has faint white outline which is just barely visible](https://user-images.githubusercontent.com/30670444/95003499-aff06c80-0594-11eb-90c9-5291258c13ee.png)

This makes it hard to navigate the docs via keyboard, and fails WCAG guidelines. 

1. [WCAG 1.4.11: Non-Text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) interactive elements need to have a focus indicator with contrast of 3:1. 
1. [WCAG 2.7.11: Focus Appearance](https://www.w3.org/TR/WCAG22/#focus-appearance-minimum) focus outlines need to be at least 1px with 3:1 contrast of the unfocused state, as well as either 3:1 against all adjacent colors or is 2px.

As well, because the current button focus style is using shadow-outline, it doesn't show up at all when using High Contrast mode. 

One option could be to create an ["outline-transparent" tailwind util like Adam mentions here](https://twitter.com/adamwathan/status/1308205581082206208?s=20). Then create a custom focus shadow util (I think by default the focus shadow uses the text color, which is why they don't necessarily show up against the gradient). 

You could also create a series of outline utils if having the outlines match your border radius isn't a primary concern.
